global_helpers/panther_oss_helpers.py

```diff
@@ -258,6 +259,26 @@ def aws_strip_role_session_id(user_identity_arn):
     return user_identity_arn
 
 
+def check_threshold(key: str,
```

Comment:
"Check threshold" makes it sound read-only, but you're changing the counter value. let's make that clear somehow. Maybe `increment_and_threshold` or `increment_and_check`